sdks/python/apache_beam/io/lyft/kafka.py

```diff
@@ -58,3 +58,57 @@ def with_bootstrap_servers(self, bootstrap_servers):
 
   def with_group_id(self, group_id):
     return self.set_kafka_consumer_property('group.id', group_id)
+
+
+class FlinkKafkaSink(PTransform):
+  """Custom transform that wraps a Flink Kafka producer - only works with the
+  portable Flink runner.
+
+  See:
+  https://ci.apache.org/projects/flink/flink-docs-stable/dev/connectors/kafka.html#kafka-producer
+
```

Comment:
I would also add an example of a usage of this class, so people will not bug you in future.